File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.py
# 선수 이름 participant
# 완주한 선수 completion

def solution(participant, completion):
    hash={}
    for p in participant:
        hash[p]=hash.get(p,0)+1
    for c in completion:
        hash[c]+=-1
    for key in hash:
        if hash[key]>0:
            return key
    answer = ''
    return answer

File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.java
import java.util.*;
class Solution {
    public String solution(String[] participant, String[] completion) {
        int i;
        Arrays.sort(participant);
        Arrays.sort(completion);
        for (i=0; i<participant.length-1; i++){
            if(!participant[i].equals(completion[i])){
                break;
            }
        }
        return participant[i];
    }
}